app/views/divisions/history.html.haml

```diff
@@ -1,15 +1,16 @@
 %style
   = Diffy::CSS
 
-- content_for :title do
-  Division Description Changes —
-  = @division.original_name
-  —
-  = formatted_date(@division.date)
+- content_for :title, "Division Description Changes"
 - set_meta_tags description: "All changes made to the description and title of the division: #{@division.name}"
 
 .page-header
   %h1= yield :title
+  %h1
```

Comment:
Thanks for flagging this @Anishka0107. I'm not sure about this, @equivalentideas what do you think?